ctru-rs/src/romfs.rs

```diff
@@ -54,10 +54,14 @@ mod tests {
     #[test]
     fn romfs_duplicate() {
         let _romfs = RomFS::init().unwrap();
+        let lock = *ROMFS_ACTIVE.lock().unwrap();
+
+        assert_eq!(lock, 1);
+
+        drop(_romfs);
```

Comment:
Does this not cause a deadlock? Wouldn't the first lock not get dropped until the end of scope?

Reply:
Yeah I named the variable wrongly. It doesn’t hold the lock (eve  though it’s called `lock`), it just holds the value by using a temporary lock.